java-engine/README.md

```diff
@@ -0,0 +1,23 @@
+# Java Bindings to Yggdrasil
+
+## Build
+
+We we gradle here:
```

Comment:
Typo? 😄

Reply:
We do do gradle here, don't see the problem

(Nice catch, absolute typo, fixed now!)